{"name": "测试用例描述：查询客户数据", "status": "failed", "statusDetails": {"message": "AssertionError: assert (['1', '董卓'],\\n ['2', '张坤'],\\n ['3', '王伟'],\\n ['4', '王伟'],\\n ['5', '宋军'],\\n ['6', '陈红'],\\n ['7', '陈红'],\\n ['8', '王丽'],\\n ['9', '王丽'],\\n ['10', '宋伟']) == (\"[['1', '董卓'], ['2', '张坤'], ['3', '王伟'], ['4', '王伟'], ['5', '宋军'], ['6', \"\\n \"'陈红'], ['7', '陈红'], ['8', '王丽'], ['9', '王丽'], ['10', '宋伟']]\")", "trace": "self = <testcase.test_one.Testmysql object at 0x000002024CDEDE80>, action = None\n\n    @allure.story('用户故事描述：查询数据')  # 模块的功能\n    @allure.description('测试描述：查询数据是否正确')\n    @allure.title('测试用例描述：查询客户数据')\n    @allure.severity(allure.severity_level.BLOCKER)\n    def test_03(self,action):\n        db = pymysql.connect(host = '192.168.0.219',port =3306,db='test',user='root',passwd='123',charset='utf8')\n        info = db.cursor()\n        info.execute('select id,username from sec_custinfo')\n        data = info.fetchall()\n        ret =[]\n        for one,two in data:\n            line = one\n            line1 = two\n            if line in excel[1]['except'] and line1 in excel[1]['except']:\n                print('数据一致')\n            ret.append([line,line1])\n        print(ret)\n        #结果：[['1', '董卓'], ['2', '张坤'], ['3', '王伟']]\n        with allure.step(\"查询\"):\n            allure.attach('select id,username from sec_custinfo',name='sql执行')\n        with allure.step(\"响应\"):\n            allure.attach('{}'.format(ret),name='返回结果')\n        with allure.step(\"断言\"):\n            assert ret[0] == ['1', '董卓']\n>           assert tuple(ret) == excel[1]['except']\nE           assert (['1', '董卓'],\\n ['2', '张坤'],\\n ['3', '王伟'],\\n ['4', '王伟'],\\n ['5', '宋军'],\\n ['6', '陈红'],\\n ['7', '陈红'],\\n ['8', '王丽'],\\n ['9', '王丽'],\\n ['10', '宋伟']) == (\"[['1', '董卓'], ['2', '张坤'], ['3', '王伟'], ['4', '王伟'], ['5', '宋军'], ['6', \"\\n \"'陈红'], ['7', '陈红'], ['8', '王丽'], ['9', '王丽'], ['10', '宋伟']]\")\n\ntest_one.py:37: AssertionError"}, "description": "测试描述：查询数据是否正确", "steps": [{"name": "查询", "status": "passed", "attachments": [{"name": "sql执行", "source": "7acd55ba-c3b6-40b8-a673-c1ae1091d516-attachment.attach"}], "start": 1651661798687, "stop": 1651661798687}, {"name": "响应", "status": "passed", "attachments": [{"name": "返回结果", "source": "4d0a67a4-dcb1-485b-8362-5dfba7834d74-attachment.attach"}], "start": 1651661798687, "stop": 1651661798687}, {"name": "断言", "status": "failed", "statusDetails": {"message": "AssertionError: assert (['1', '董卓'],\\n ['2', '张坤'],\\n ['3', '王伟'],\\n ['4', '王伟'],\\n ['5', '宋军'],\\n ['6', '陈红'],\\n ['7', '陈红'],\\n ['8', '王丽'],\\n ['9', '王丽'],\\n ['10', '宋伟']) == (\"[['1', '董卓'], ['2', '张坤'], ['3', '王伟'], ['4', '王伟'], ['5', '宋军'], ['6', \"\\n \"'陈红'], ['7', '陈红'], ['8', '王丽'], ['9', '王丽'], ['10', '宋伟']]\")\n", "trace": "  File \"C:\\Users\\Administrator\\.jenkins\\workspace\\psimsendemail\\untitled\\testcase\\test_one.py\", line 37, in test_03\n    assert tuple(ret) == excel[1]['except']\n"}, "start": 1651661798687, "stop": 1651661798688}], "start": 1651661798684, "stop": 1651661798688, "uuid": "010b8dcc-964c-4c88-a673-b64ad736f738", "historyId": "e80dcb360667126e504de2bd04d2fb61", "testCaseId": "673d513ad150bc91b8f870520ad0cdeb", "fullName": "test_one.Testmysql#test_03", "labels": [{"name": "epic", "value": "*************测试描述*************"}, {"name": "severity", "value": "blocker"}, {"name": "feature", "value": "数据库"}, {"name": "story", "value": "用户故事描述：查询数据"}, {"name": "suite", "value": "test_one"}, {"name": "subSuite", "value": "Testmysql"}, {"name": "host", "value": "MS-SLSXLICKSUSF"}, {"name": "thread", "value": "2720-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_one"}]}